week2/2-website/css/style.css

```diff
@@ -5,6 +5,194 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+header {
+    background-color: white;
+    position: fixed;
+    top: 0;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+
+#logo {
+    display: flex;
+    justify-content: center;
+    width: 43vw;
+}
+#header-img {
+    width: 40px;
+    height: 40px;  
+}
+nav {
+    width: 57vw;
```

Comment:
same

Reply:
Consider adding another wrapper, around the logo and nav and give it a max-width of around 675px.
...and center aligned. This will allow you to push the nav and logos out as far as they can go using `justify-content:space-between`